ascon/Cargo.toml

```diff
@@ -1,34 +1,37 @@
 [package]
 name = "ascon"
-version = "0.2.0"
-description = """
-Pure Rust implementation of Ascon, a family of authenticated encryption and
-hashing algorithms designed to be lightweight and easy to implement
-"""
-authors = ["quininer kel <quininer@live.com>", "RustCrypto Developers"]
+version = "0.3.0-pre"
```

Comment:
Have you forgot to rename the crate to `ascon-core`?

Reply:
If I understood @tarcieri correctly, the intention was to move the functionality of `ascon-core` into `ascon`. The AEAD implementation will later on be provided by `ascon-aead`. But I'm also happy to keep the name as `ascon-core` if that is what @RustCrypto prefers.

Reply:
Yeah, I'd suggest using `ascon`